portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,48 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random picture with its description to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function getRandomImage() {
+  /** !Array<Objects> */
+  /* @type {{img_src: string, description: string }}  */
+  const galleryItems = [
+    {
+      img_src: "food_picture.jpg",
```

Comment:
JavaScript uses camel-case convention for most variables and attributes. So instead of img_src, please use imgSrc